libs/x-content/impl/build.gradle

```diff
@@ -10,7 +10,7 @@ apply plugin: 'elasticsearch.java'
 
 archivesBaseName = "x-content-impl"
 
-String jacksonVersion = "2.14.2"
+String jacksonVersion = "2.15.0"
```

Comment:
Is there a reason we don't just use what's in `version.properties` here?

Reply:
Yes, we want x-content to be independent because we have had situations where we want to update this more quickly. I don't think we should have jackson in version.properties anymore at all, but it's been difficult to get momentum to remove it from there.